views/chitter/home.erb

```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html>
```

Comment:
Generally really liking your layout for your views - well formed, easy to follow and understand.